libs/remixd/src/services/hardhatClient.ts

```diff
@@ -54,7 +54,7 @@ export class HardhatClient extends PluginClient {
         const errMsg = '[Hardhat Compilation]: Cannot compile in read-only mode'
         return reject(new Error(errMsg))
       }
-      const cmd = `npx hardhat compile --config ${configPath}`
+      const cmd = utils.normalizePath(`./node_modules/.bin/hardhat compile --config ${utils.normalizePath(configPath)}`)
```

Comment:
this will only work if node is in the path that electron has when running standalone. 
[Hardhat Compilation]: env: node: No such file or directory